js/schema/style.json

```diff
@@ -29,6 +29,17 @@
     "size": {
```

Comment:
What I meant is that I think we should use the fixed `size` options from here (`small, medium, large`) instead of creating new button-width and button-height.
At rendering time we can define what small, medium, large convert to in pixels.